packages/shield-controller/src/backend.ts

```diff
@@ -193,15 +210,26 @@ export class ShieldRemoteBackend implements ShieldBackend {
       coverageId,
     };
 
+    const abortController = this.#assignNewAbortController(configs.requestId);
```

Comment:
Is there a reason why you create this here and not within the Promise?

(We could probably move all things from the outer context into the Promise. Might be cleaner overall.)